chart/templates/storageclass.yaml

```diff
@@ -2,6 +2,8 @@ kind: StorageClass
 apiVersion: storage.k8s.io/v1
 metadata:
   name: {{ .Release.Name }}-single-replica
+  labels:
```

Comment:
do we need it for the sc?

Reply:
Not really. Just putting it on all of the mayastor components.